modules/_labs/drawer/react/lib/DrawerHeader.tsx

```diff
@@ -0,0 +1,107 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import {typeColors} from '@workday/canvas-colors-web';
+import {colors, spacing, H4, CanvasColor} from '@workday/canvas-kit-react-core';
+import {IconButton, IconButtonVariant} from '@workday/canvas-kit-react-button';
+import {xIcon} from '@workday/canvas-system-icons-web';
+
+export interface DrawerHeaderProps extends React.HTMLAttributes<HTMLDivElement> {
+  /**
+   * String to add a title to a Drawer
+   */
+  title?: string;
+  /**
+   * Callback to handle closing the Drawer
+   */
+  onClose: () => void;
+  /**
+   * String to add aria-label to the icon button
+   */
+  iconLabel: string;
+  /**
+   * The background color in which the Drawer header will be
+   */
+  headerColor: CanvasColor | string;
+  /**
+   * Changes the border color to match something close to the header background color
+   */
+  borderColor: CanvasColor | string;
+  /**
+   * If the background of the header is dark, this will switch the icon so that it is white
+   */
+  showInverseButton?: boolean;
```

Comment:
Nit: Both this and `titleColor` are impacted by the same thing (headerColor). Why not just have an `inverse?: boolean` flag that changes the button type and changes the text color to white?

Reply:
ah yep, that makes sense!